medcat/meta_cat.py

```diff
@@ -356,6 +356,17 @@ def load(cls, save_dir_path: str, config_dict: Optional[Dict] = None) -> "MetaCA
         meta_cat.model.load_state_dict(torch.load(model_save_path, map_location=device))
 
         return meta_cat
+    
+    def get_ents(self, doc: Doc) -> List[Span]:
```

Comment:
The return type should probably be `Iterable[Span]` so as to allow both lists and tuples. The result is just iterated over - it does not need to be changeable (i.e doesn't need to be a `list` and can indeed be a `tuple` - which is what `mypy` says happens on line 369)